### ISSUE TITLE
Build with output path to network share drive fails

### ISSUE THREAD

**Issue by fehrinjt** [angular/angular-cli#8030]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
@angular/cli: 1.4.5
node: 6.9.5
os: win32 x64
@angular/common: 4.4.4
@angular/compiler: 4.4.4
@angular/core: 4.4.4
@angular/forms: 4.4.4
@angular/http: 4.4.4
@angular/platform-browser: 4.4.4
@angular/platform-browser-dynamic: 4.4.4
@angular/router: 4.4.4
@angular/cli: 1.4.5
@angular/compiler-cli: 4.4.4
@angular/language-service: 4.4.4
typescript: 2.3.4

### Repro steps.
Our build process copies the project to a server using the --output-path command. Recent versions of the CLI have broken this. All builds are done from windows machines.
Example project can be found here: [New CLI Project](https://github.com/fehrinjt/CLIBugTest). This is a newly created CLI project.
To get the error:
ng build --op \\\\serverName\\apps\\CliBugTest\\ --dev
OR
npm run publish
NOTE: serverName must point to an actual share drive/server name or a different error is given

### The log given by the failure.
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.output.path: The provided value "\\\\serverName\\apps\\CliBugTest" is not an absolute path!
WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.output.path: The provided value "\\\\serverName\\apps\\CliBugTest" is not an absolute path!
    at webpack (C:\SourceControl\GIT\CLIBugTest\node_modules\webpack\lib\webpack.js:19:9)
    at Class.run (C:\SourceControl\GIT\CLIBugTest\node_modules\@angular\cli\tasks\build.js:28:33)
    at Class.run (C:\SourceControl\GIT\CLIBugTest\node_modules\@angular\cli\commands\build.js:204:26)
    at resolve (C:\SourceControl\GIT\CLIBugTest\node_modules\@angular\cli\ember-cli\lib\models\command.js:261:20)
    at Class.validateAndRun (C:\SourceControl\GIT\CLIBugTest\node_modules\@angular\cli\ember-cli\lib\models\command.js:240:12)
    at Promise.resolve.then.then (C:\SourceControl\GIT\CLIBugTest\node_modules\@angular\cli\ember-cli\lib\cli\cli.js:140:24)

### Desired functionality.
Build should output to network share.

### Mention any other details that might be useful.
This method worked with CLI version 1.0.0.

**Comment by clydin** [comment]
Webpack does not consider the supplied value an absolute path.  The check within wepback is here: https://github.com/webpack/webpack/blob/master/schemas/ajv.absolutePath.js#L20

As a workaround, the network path can be mapped to a drive letter.

**Comment by bmtheo** [comment]
Hello @mgechev,

Yes we still have the problem, see details below



**Version**

Angular CLI: 7.3.6
Node: 10.15.1
OS: win32 x64
Angular: 7.2.9
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.13.3
@angular-devkit/build-angular     0.13.3
@angular-devkit/build-optimizer   0.13.3
@angular-devkit/build-webpack     0.13.3
@angular-devkit/core              7.3.3
@angular-devkit/schematics        7.3.6
@angular/cdk                      7.3.4
@angular/cli                      7.3.6
@angular/flex-layout              7.0.0-beta.23
@angular/material                 7.3.4
@ngtools/webpack                  7.3.3
@schematics/angular               7.3.6
@schematics/update                0.13.6
rxjs                              6.4.0
typescript                        3.2.4
webpack                           4.29.0
    
**Error**

Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.output.path: The provided value "\\\\gew12file1\\prog\\ordi\\newopaleweb" is not an absolute path!
   -> The output directory as **absolute path** (required).
WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.output.path: The provided value "\\\\gew12file1\\prog\\ordi\\newopaleweb" is not an absolute path!
   -> The output directory as **absolute path** (required).
    at webpack (C:\ordi\dev\web\OpaleWeb\node_modules\webpack\lib\webpack.js:31:9)
    at WebpackBuilder.createWebpackCompiler (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-webpack\src\webpack\index.js:20:16)
    at Observable.rxjs_1.Observable.obs [as _subscribe] (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-webpack\src\webpack\index.js:24:42)
    at Observable._trySubscribe (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-webpack\node_modules\rxjs\internal\Observable.js:44:25)
    at Observable.subscribe (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-webpack\node_modules\rxjs\internal\Observable.js:30:22)
    at C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-angular\node_modules\rxjs\internal\util\subscribeToObservable.js:10:20
    at Object.subscribeToResult (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-angular\node_modules\rxjs\internal\util\subscribeToResult.js:10:45)
    at MergeMapSubscriber._innerSub (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-angular\node_modules\rxjs\internal\operators\mergeMap.js:82:29)
    at MergeMapSubscriber._tryNext (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-angular\node_modules\rxjs\internal\operators\mergeMap.js:76:14)
    at MergeMapSubscriber._next (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-angular\node_modules\rxjs\internal\operators\mergeMap.js:59:18)
    at MergeMapSubscriber.Subscriber.next (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-angular\node_modules\rxjs\internal\Subscriber.js:67:18)
    at MergeMapSubscriber.notifyNext (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-angular\node_modules\rxjs\internal\operators\mergeMap.js:92:26)
    at InnerSubscriber._next (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-angular\node_modules\rxjs\internal\InnerSubscriber.js:28:21)
    at InnerSubscriber.Subscriber.next (C:\ordi\dev\web\OpaleWeb\node_modules\@angular-devkit\build-angular\node_modules\rxjs\internal\Subscriber.js:67:18)
    at SafeSubscriber.__tryOrUnsub (C:\ordi\dev\web\OpaleWeb\node_modules\@angular\cli\node_modules\rxjs\internal\Subscriber.js:209:16)
    at SafeSubscriber.next (C:\ordi\dev\web\OpaleWeb\node_modules\@angular\cli\node_modules\rxjs\internal\Subscriber.js:147:22)

[version.txt](https://github.com/angular/angular-cli/files/2970668/version.txt)
[builderror.txt](https://github.com/angular/angular-cli/files/2970680/builderror.txt)

**Comment by philliprobinson** [comment]
A way to get around this for now is to map to the network (e.g. Z: mapping to \\serverName\apps\CliBugTest\ as described in your problem). You can then specify Z: as your outputPath.

**Comment by clydin** [comment]
This should be corrected as of Angular CLI 9.1.0 which updated Webpack to 4.42.0 (contains this [fix](https://github.com/webpack/webpack/commit/d8c74b68d521c2d9632356e4435a8feb4f3229f3)).
If you are still encountering this issue with an updated project, please let us know.
